components/gateway/internal/gqlschema/labels.go

```diff
@@ -0,0 +1,100 @@
+package gqlschema
```

Comment:
If we want to have all custom defined scalars in single file, I think we should rename the file to something like `types.go` or `scalars.go`.